cerberus/cerberus.py

```diff
@@ -701,6 +722,13 @@ def _validate_allof(self, allof, field, value):
         if not valid:
             self._error(field, errorstack)
 
+    def _validate_rename(self, rename, field, value):
+        if isinstance(rename, _str_type):
```

Comment:
You're right, I'll add the check there and will change the type to `collections.Hashable`.